<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecodeDesigner</class>
 <widget class="QMainWindow" name="MinecodeDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正黑体简体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Minecode Command Layer Designer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDockWidget::title{
text-align: left center;
background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.494, y2:0.0284091, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(240, 240, 240, 255));
}

QToolBox::tab{
    border: 0px;
	background-color: rgb(200, 200, 200);
	color: rgb(45,45,45);
}

QMenu{
    background-color: rgb(245, 245, 245);
    color: rgb(75, 75, 75);
}

QMenu::separator{
    height: 1px;
    background-color: rgb(200, 200, 200);
    margin-left: 3px;
    margin-right: 3px;
    margin-top: 2px;
    margin-bottom: 2px;
}

QMenu::item:selected {
	color: black;
	background-color: rgb(225, 245, 254);
}

QMenu::item{
    border: 0px solid;
    padding:6px 30px;
}

QMenu::right-arrow {
    image: url(:/ToolIcon/src/ToolBar/arrow-right.png);
}

QMenu::left-arrow {
    image: url(:/ToolIcon/src/ToolBar/arrow-left.png);
}

QMenu::icon{
    padding: 3px 3px 3px 3px;
    border: 0px solid;
}

QMenu::indicator{
    border: 0px solid;
}

QMenu::indicator:selected{
    border: 0px solid;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QStackedWidget" name="MultiViewEditor">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>Edit (&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionMenu_Layer"/>
     <addaction name="actionSpace_2"/>
     <addaction name="separator"/>
     <addaction name="actionCommand_2"/>
     <addaction name="actionRedstoneDelay"/>
     <addaction name="actionScoreboradObject"/>
     <addaction name="actionSiganl_2"/>
     <addaction name="actionSlot_2"/>
    </widget>
    <widget class="QMenu" name="menuLayerEdit">
     <property name="title">
      <string>Layer Edit</string>
     </property>
     <addaction name="actionPrevious_Layer"/>
     <addaction name="actionNext_Layer"/>
    </widget>
    <widget class="QMenu" name="menuCommand_Block">
     <property name="title">
      <string>Command Block</string>
     </property>
     <addaction name="actionPulse"/>
     <addaction name="actionRepeat"/>
     <addaction name="actionChain"/>
    </widget>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="actionLoad_From_New_File"/>
     <addaction name="actionLoad_From_Project"/>
     <addaction name="actionLoad_From_Existing_File"/>
     <addaction name="actionLoad_From_Existing_Project"/>
     <addaction name="separator"/>
     <addaction name="actionLoad_Minecraft_Texture_Pack"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menuLayerEdit"/>
    <addaction name="menuCommand_Block"/>
    <addaction name="menuLoad"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>View (&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Dimension</string>
     </property>
     <addaction name="action2D"/>
     <addaction name="action3D"/>
    </widget>
    <widget class="QMenu" name="menuMinecraft_Texture_Pack">
     <property name="title">
      <string>Minecraft Texture Pack</string>
     </property>
     <addaction name="action_Default"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menuMinecraft_Texture_Pack"/>
   </widget>
   <addaction name="menu_E"/>
   <addaction name="menu_V"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="action3DView"/>
   <addaction name="action2DView"/>
   <addaction name="actionLayer"/>
   <addaction name="actionSpace"/>
   <addaction name="separator"/>
   <addaction name="actionCommand"/>
   <addaction name="actionDelay"/>
   <addaction name="actionScoreboard"/>
   <addaction name="actionSiganl"/>
   <addaction name="actionSlot"/>
   <addaction name="separator"/>
   <addaction name="actionPreviousLayer"/>
   <addaction name="actionNextLayer"/>
   <addaction name="separator"/>
   <addaction name="actionNormalCmd"/>
   <addaction name="actionRepeatCmd"/>
   <addaction name="actionChainCmd"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Tool"/>
   <addaction name="actionHand_Tool"/>
   <addaction name="actionZoom_Tool"/>
   <addaction name="actionRotate_Tool"/>
   <addaction name="actionCamera"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="font">
    <font>
     <family>方正黑体简体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Toolbox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>方正黑体简体</family>
        </font>
       </property>
       <property name="inputMask">
        <string>Filter</string>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="font">
        <font>
         <family>方正黑体简体</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>133</width>
          <height>471</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Command Block</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>133</width>
          <height>471</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Redstone Tools</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>133</width>
          <height>471</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Signal/Slot Object</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>133</width>
          <height>471</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Layer &amp;&amp; Space</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="font">
    <font>
     <family>方正黑体简体</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>Object Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="objectViewer">
       <property name="font">
        <font>
         <family>方正黑体简体</family>
        </font>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="font">
    <font>
     <family>方正黑体简体</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>Command Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="cmdEditor">
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action3DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/3DView.png</normaloff>:/ToolIcon/src/ToolBar/3DView.png</iconset>
   </property>
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
  <action name="action2DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/2DView.png</normaloff>:/ToolIcon/src/ToolBar/2DView.png</iconset>
   </property>
   <property name="text">
    <string>2D View</string>
   </property>
  </action>
  <action name="actionLayer">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Layer.png</normaloff>:/ToolIcon/src/ToolBar/Layer.png</iconset>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </action>
  <action name="actionSpace">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Space.png</normaloff>:/ToolIcon/src/ToolBar/Space.png</iconset>
   </property>
   <property name="text">
    <string>Space</string>
   </property>
  </action>
  <action name="actionCommand">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Command.png</normaloff>:/ToolIcon/src/ToolBar/Command.png</iconset>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </action>
  <action name="actionSiganl">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Signal.png</normaloff>:/ToolIcon/src/ToolBar/Signal.png</iconset>
   </property>
   <property name="text">
    <string>Signal</string>
   </property>
  </action>
  <action name="actionSlot">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Slot.png</normaloff>:/ToolIcon/src/ToolBar/Slot.png</iconset>
   </property>
   <property name="text">
    <string>Slot</string>
   </property>
  </action>
  <action name="actionPreviousLayer">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/PreviousLayer.png</normaloff>:/ToolIcon/src/ToolBar/PreviousLayer.png</iconset>
   </property>
   <property name="text">
    <string>Previous Layer</string>
   </property>
  </action>
  <action name="actionNextLayer">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/NextLayer.png</normaloff>:/ToolIcon/src/ToolBar/NextLayer.png</iconset>
   </property>
   <property name="text">
    <string>Next Layer</string>
   </property>
  </action>
  <action name="actionDelay">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Delay.png</normaloff>:/ToolIcon/src/ToolBar/Delay.png</iconset>
   </property>
   <property name="text">
    <string>Redstone Delay</string>
   </property>
  </action>
  <action name="actionNormalCmd">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/NormalCmd.png</normaloff>:/ToolIcon/src/ToolBar/NormalCmd.png</iconset>
   </property>
   <property name="text">
    <string>Impulse Mode</string>
   </property>
  </action>
  <action name="actionRepeatCmd">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/RepeatCmd.png</normaloff>:/ToolIcon/src/ToolBar/RepeatCmd.png</iconset>
   </property>
   <property name="text">
    <string>Repeat Mode</string>
   </property>
  </action>
  <action name="actionChainCmd">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/ChainCmd.png</normaloff>:/ToolIcon/src/ToolBar/ChainCmd.png</iconset>
   </property>
   <property name="text">
    <string>Chain Mode</string>
   </property>
  </action>
  <action name="actionScoreboard">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ToolIcon/src/ToolBar/Scoreboard.png</normaloff>:/ToolIcon/src/ToolBar/Scoreboard.png</iconset>
   </property>
   <property name="text">
    <string>Scoreboard</string>
   </property>
  </action>
  <action name="action2D">
   <property name="text">
    <string>2D</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionMenu_Layer">
   <property name="text">
    <string>Layer</string>
   </property>
  </action>
  <action name="actionSpace_2">
   <property name="text">
    <string>Space</string>
   </property>
  </action>
  <action name="actionCommand_2">
   <property name="text">
    <string>Command</string>
   </property>
  </action>
  <action name="actionRedstoneDelay">
   <property name="text">
    <string>Redstone Delay</string>
   </property>
  </action>
  <action name="actionScoreboradObject">
   <property name="text">
    <string>Scoreborad Object</string>
   </property>
  </action>
  <action name="actionSiganl_2">
   <property name="text">
    <string>Siganl</string>
   </property>
  </action>
  <action name="actionSlot_2">
   <property name="text">
    <string>Slot</string>
   </property>
  </action>
  <action name="actionPrevious_Layer">
   <property name="text">
    <string>Previous Layer</string>
   </property>
  </action>
  <action name="actionNext_Layer">
   <property name="text">
    <string>Next Layer</string>
   </property>
  </action>
  <action name="actionPulse">
   <property name="text">
    <string>Pulse</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionChain">
   <property name="text">
    <string>Chain</string>
   </property>
  </action>
  <action name="actionLoad_From_New_File">
   <property name="text">
    <string>Load From New File</string>
   </property>
  </action>
  <action name="actionLoad_From_Project">
   <property name="text">
    <string>Load From New Project</string>
   </property>
  </action>
  <action name="actionLoad_From_Existing_File">
   <property name="text">
    <string>Load From Existing File</string>
   </property>
  </action>
  <action name="actionLoad_From_Existing_Project">
   <property name="text">
    <string>Load From Existing Project</string>
   </property>
  </action>
  <action name="actionLoad_Minecraft_Texture_Pack">
   <property name="text">
    <string>Load Minecraft Texture Pack</string>
   </property>
  </action>
  <action name="action_Default">
   <property name="text">
    <string>( Default )</string>
   </property>
  </action>
  <action name="actionSelect_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/Cursor/cursor.png</normaloff>:/Cursor/src/Cursor/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Select Tool</string>
   </property>
  </action>
  <action name="actionHand_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/Cursor/hand.png</normaloff>:/Cursor/src/Cursor/hand.png</iconset>
   </property>
   <property name="text">
    <string>Hand Tool</string>
   </property>
  </action>
  <action name="actionZoom_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/Cursor/zoom.png</normaloff>:/Cursor/src/Cursor/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/ActionIcon/undo.png</normaloff>:/Cursor/src/ActionIcon/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/ActionIcon/redo.png</normaloff>:/Cursor/src/ActionIcon/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionRotate_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/Cursor/rotate.png</normaloff>:/Cursor/src/Cursor/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Tool</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Cursor/src/Cursor/camera.png</normaloff>:/Cursor/src/Cursor/camera.png</iconset>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
